plugins/newsdownloader.koplugin/main.lua

```diff
@@ -152,11 +152,12 @@ function NewsDownloader:loadConfigAndProcessFeeds()
 
     local unsupported_feeds_urls = {}
 
+    local total_feed_entries = table.getn(feed_config)
     for idx, feed in ipairs(feed_config) do
         local url = feed[1]
         local limit = feed.limit
         if url and limit then
-            info = InfoMessage:new{ text = T(_("Processing: %1"), url) }
+            info = InfoMessage:new{ text = T(_("Processing %1/%2: %3"), idx, total_feed_entries, url) }
```

Comment:
Probably better to use a newline instead of a colon.

Reply:
my all feeds are so long I've never seen it in 1 line - it's new line automatic;)
but i'll commit change.